src/core/account.service.ts

```diff
@@ -116,22 +116,6 @@ export class AccountService {
     logger.warn("Don't have controller pk in db");
   }
 
-  async activeController(address: string): Promise<Controller> {
```

Comment:
why delete this?  is this listening chain event ?

Reply:
We don't need to listen chain event, the function will fetch controller from contract directly for each query